src/Pos/Util/UserSecret.hs

```diff
@@ -1,8 +1,13 @@
+{-# LANGUAGE CPP             #-}
 {-# LANGUAGE TemplateHaskell #-}
 
 -- | Secret key file storage and management functions based on file
 -- locking.
 
+#if !defined(mingw32_HOST_OS)
+#define POSIX
+#endif
```

Comment:
Such redefinitions only introduce confusion in the code, are they really necessary?